### ISSUE TITLE
Winter month feature in openrct2

### ISSUE THREAD

**Issue by harrysbird** [OpenRCT2/OpenRCT2#5499]
I love this game but we should add winter month feature so we will have a winter amusement park.

Before: March to October
After: January to December

I would to have this feature.

**Comment by e-foley** [comment]
How would this affect scenario objectives?

**Comment by jensj12** [comment]
The whole codebase is based on years consisting of 8 months. This feature could be implemented by 'scaling' the date shown, but that will make false messages (4 weeks left to fix park rating when you still have 6) and ducks would come and go in different seasons.

Apart from the technical details, how would it make the game better?

**Comment by IntelOrca** [comment]
Duplicate of #1634

**Comment by Gymnasiast** [comment]
Indeed a duplicate of #1634. The reasons in there still apply as well.
